package.json

```
@@ -18,17 +18,18 @@
     "build": "lerna run build",
     "clean": "lerna run clean",
     "build:tsc": "tsc -b packages/**/tsconfig.build.json",
-    "lint": "eslint . --fix --quiet",
-    "lint:check": "eslint . --quiet",
+    "lint": "lerna run lint",
+    "lint:ci": "yarn format:check && yarn lint:check && yarn packageJsonLint",
+    "lint:check": "lerna run lint:check",
     "lint-staged": "lint-staged",
     "lerna": "lerna",
     "packageJsonLint": "npmPkgJsonLint ./packages",
-    "publish-npm": "lerna publish --conventional-commits --exact",
-    "publish-manual-prerelease": "lerna publish prerelease --conventional-commits --exact",
-    "publish-prerelease": "yarn lerna publish --preid development --skip-git --yes --canary",
+    "publish-npm": "yarn lerna publish --conventional-commits --exact --message \"chore(release): publish %s\" --no-changelog --yes",
+    "publish-manual-prerelease": "lerna publish prerelease --conventional-commits --conventional-prerelease --exact --no-git-tag-version --no-push --preid next --no-verify-access --dist-tag next",
+    "publish-prerelease": "FORCE_PUBLISH=$($(yarn bin)/lerna changed --json | jq '. | map(.name) | join (\",\")' -r) && echo $FORCE_PUBLISH && $(yarn bin)/lerna publish --conventional-commits --conventional-prerelease --exact --no-git-tag-version --no-push --preid next --no-verify-access --dist-tag next --yes --force-publish=${FORCE_PUBLISH}",
```

I don't think we need the `$(yarn bin)`

```suggestion
    "publish-npm": "lerna publish --conventional-commits --exact --message \"chore(release): publish %s\" --no-changelog --yes",
    "publish-manual-prerelease": "lerna publish prerelease --conventional-commits --conventional-prerelease --exact --no-git-tag-version --no-push --preid next --no-verify-access --dist-tag next",
    "publish-prerelease": "FORCE_PUBLISH=$(lerna changed --json | jq '. | map(.name) | join (\",\")' -r) && echo $FORCE_PUBLISH && lerna publish --conventional-commits --conventional-prerelease --exact --no-git-tag-version --no-push --preid next --no-verify-access --dist-tag next --yes --force-publish=${FORCE_PUBLISH}",
```

---

Makes sense!!